doc/api/fs.md

```diff
@@ -959,7 +966,7 @@ The "not recommended" examples above check for existence and then use the
 file; the "recommended" examples are better because they use the file directly
 and handle the error, if any.
 
-In general, check for the existence of a file only if the file won’t be
+In general, check for the existence of a file only if the file won't be
```

Comment:
@refack We have a lint rule against BOMs. I’m not sure whether the linter catches that for markdown files, but generally, no, please don’t add those.

In this case it doesn’t matter much, it just makes small lines a bit uglier, but if you don’t want to get condescending looks from everyone outside the American bubble for being ASCII-only, maybe fix your editor config instead. ;)